javascript/javascript1/week1/myscripts.js

```diff
@@ -0,0 +1,81 @@
+// Task 1
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+// Task 2
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let dogYear = dogYearFuture - dogYearOfBirth + 60;
+let shouldShowResultInDogYear = true;
+
+if (shouldShowResultInDogYear == true) {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture
+  );
+} else {
+  let humanYear = dogYear - 60;
```

Comment:
`const humanYear = dogYear * 7;`

Reply:
and you can also put it next to other defined variables, after `dogYear` - line 11, and before `shouldShowResultInDogYear`